lib/Role/Basic.pm

```diff
@@ -615,6 +620,10 @@ Returns true if the class or role consumes a role of the given name:
 
 Every role "DOES" itself.
 
+=item * C<ROLES>
+
+Returns the list of all roles consumed by the class or role, including itself.
```

Comment:
`s/the list/an unordered list/`

This seems like a minor thing, but I'm trying to be more clear in my docs nowadays, to avoid any possibility that people will rely on guarantees I haven't made 😃